packages/documentation/docs/controls/charts/overview.md

```diff
@@ -6,33 +6,66 @@ import Playground from '@site/src/components/PlaygroundV3';
 # Basics
 
 Siemens Industrial Experience provides a theme for the popular chart library [ECharts](https://echarts.apache.org/handbook/en/get-started).
-This lets you harness the power of ECharts with seamless integration into the Siemens Industrial Experience styleguide.
+This lets you harness the power of ECharts with seamless integration into the Siemens Industrial Experience design system.
 
 <div className="siemens-brand-section">
-
 ECharts is a third party library distributed under [Apache License 2.0](https://www.apache.org/licenses).
-
 </div>
 
+![Chart usage guide](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3532-4181&t=MD9MvUCkoIcmSi8H-4)
+
+## Attributes
+
+| Name           | Description                                                                                                                                                                                   |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Axes**       | Most charts have a horizontal (x-axis) and a vertical (y-axis) axis. These axes are labeled to indicate what data they represent.                                                             |
+| **Scale**      | Each axis has a scale, often marked with periodic graduations and numerical or categorical indications.                                                                                       |
```

Comment:
Is there a simpler way to describe this?
Plus, I think it should be "every axis"